### ISSUE TITLE
the composer package version has a version conflict between codeception/phpunit-wrapper and symfony/event-dispatcher

### ISSUE THREAD

**Issue by aslackers** [Codeception/Codeception#5807]
```shell
Uncaught TypeError: Argument 1 passed to Symfony\Component\EventDispatcher\EventDispatcher::dispatch() must be an object, string given, called in app-path/vendor/codeception/phpunit-wrapper/src/Listener.php on line 135 and defined in app-path/vendor/symfony/event-dispatcher/EventDispatcher.php:48
```
my `composer.json`:

```php
...
"require-dev": {
        "yiisoft/yii2-debug": "~2.1.0",
        "yiisoft/yii2-gii": "~2.1.0",
        "yiisoft/yii2-faker": "~2.0.0",
        "codeception/codeception": "^4.0",
        "codeception/module-asserts": "^1.0",
        "codeception/module-yii2": "^1.0",
        "codeception/module-filesystem": "^1.0",
        "phpunit/phpunit": "~5.7.27 || ~6.5.5",
        "codeception/verify": "~0.5.0 || ~1.1.0",
        "symfony/browser-kit": ">=2.7 <=4.2.4"
    },
...
```
It's because in the `codeception/phpunit-wrapper/src/Listener.php`:
```php
$this->dispatcher->dispatch('suite.end', new SuiteEvent($suite));
```
but in `symfony/event-dispatcher` it's defined:
```php
public function dispatch(object $event, string $eventName = null): object;
```

**Comment by Naktibalda** [comment]
Currently this error only happens if you have PHPUnit 5 or 6 and Symfony/event-dispatcher 5.0 installed.

This (Yii2 app) is a good enough reason to copy Symfony 5 support to 6.5 branch of phpunit-wrapper.

**Comment by Naktibalda** [comment]
I released phpunit-wrapper 6.8.0 to make PHPUnit 6.5 compatible with Symfony 5
and phpunit-wrapper 6.1.3 to make PHPUnit 5.7 compatible with Symfony 5.

Also I raised pull request to remove phpunit version constraint from yii2-app-advanced: https://github.com/yiisoft/yii2-app-advanced/pull/455
I recommend you do the same.
